packages/calling/src/Contacts/contactFixtures.ts

```diff
@@ -272,42 +272,40 @@ export const mockContactGroupListTwo = [
   },
 ];
 
-export const mockDSSResponse = [
-  {
-    additionalInfo: {
-      created: '2022-08-05T02:51:46.055Z',
-      department: '123029217',
-      extLinkedAccts: [{providerID: 'cisco.webex.com', accountGUID: '500802287', status: 'active'}],
-      firstName: 'Emily',
-      identityManager: {
-        displayName: 'Robert Langdon',
-        managerId: '9d0fce00-95b2-435f-99d1-b6b44759fbdc',
-      },
-      jobTitle: 'Software Engineer',
-      lastName: 'Nagakawa',
-      modified: '2023-03-03T13:37:03.196Z',
-      nickName: 'Emily',
-      userName: 'emikawa2@cisco.com',
+export const mockDSSResponse = {
```

Comment:
mockDSSResponse is no longer an array because the lookup returns a single search result object.